welly/well.py

```diff
@@ -161,11 +176,10 @@ def from_lasio(cls,
                    req=None,
                    alias=None,
                    fname=None,
-                   index=None,
-                   ):
+                   index=None):
         """
         Constructor. If you already have the lasio object, then this makes a
-        well object from it.
+        well object from it. Will be deprecated but through this workaround it still functions for now.
```

Comment:
hm, do we want to deprecate the functionality?

There is a case to be made to load from lasio object (in-memory) vs loading from LAS file (hard drive).

Whether we should have two methods is a good question, but it seems reasonable.

Long-term, we could even try moving to a single function for "from path" and a single function for "from file", which auto-detects format?